src/qt/forms/overviewpage.ui

```diff
@@ -352,40 +362,130 @@
                          </property>
                         </widget>
                        </item>
-                       <item row="3" column="2">
-                        <widget class="QLabel" name="labelWatchImmature">
+                       <item row="2" column="3">
+                        <spacer name="horizontalSpacer_2">
+                         <property name="orientation">
+                          <enum>Qt::Horizontal</enum>
+                         </property>
+                         <property name="sizeHint" stdset="0">
+                          <size>
+                           <width>40</width>
+                           <height>20</height>
+                          </size>
+                         </property>
+                        </spacer>
+                       </item>
+                      </layout>
+                     </item>
+                    </layout>
+                   </widget>
+                  </item>
+                  <item>
+                   <widget class="QFrame" name="frame_Balances">
+                    <property name="frameShape">
+                     <enum>QFrame::StyledPanel</enum>
+                    </property>
+                    <property name="frameShadow">
+                     <enum>QFrame::Raised</enum>
+                    </property>
+                    <layout class="QVBoxLayout" name="verticalLayout_4">
+                     <item>
+                      <layout class="QHBoxLayout" name="horizontalLayout_4">
+                       <item>
+                        <widget class="QLabel" name="label_5">
+                         <property name="minimumSize">
+                          <size>
+                           <width>0</width>
+                           <height>20</height>
+                          </size>
+                         </property>
                          <property name="font">
                           <font>
+                           <pointsize>12</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                           </font>
                          </property>
-                         <property name="cursor">
-                          <cursorShape>IBeamCursor</cursorShape>
+                         <property name="text">
+                          <string>PIVX Balances</string>
                          </property>
-                         <property name="toolTip">
-                          <string>Staked or masternode rewards in watch-only addresses that has not yet matured</string>
+                        </widget>
+                       </item>
+                       <item>
+                        <widget class="QLabel" name="labelPIVPercent">
+                         <property name="text">
+                          <string>0 %</string>
```

Comment:
should probably add `notr="true"` to the `<string>` element here.

Reply:
I am pretty sure this was not everywhere before. Will add of course.